Formula/brooklyn-client.rb

```diff
@@ -0,0 +1,40 @@
+require "language/go"
+
+class BrooklynClient < Formula
+  desc "Client for controlling Apache Brooklyn from the command-line"
+  homepage "https://github.com/apache/brooklyn-client"
+  url "https://github.com/apache/brooklyn-client.git",
+      :tag => "rel/apache-brooklyn-0.9.0",
+      :revision => "bc8593a933fcb76327ae4a511643e39d25a87ba2"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/codegangsta/cli" do
+    url "https://github.com/codegangsta/cli.git",
+      :revision => "5db74198dee1cfe60cf06a611d03a420361baad6"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git",
+      :revision => "1f22c0103821b9390939b6776727195525381532"
+  end
+
+  def install
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+    ENV["GOPATH"] = buildpath
+
+    brooklyn_client_path = buildpath/"src/github.com/apache/brooklyn-client/"
+    brooklyn_client_path.install Dir["*"]
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/apache/brooklyn-client/br" do
+      system "go", "build"
+      bin.install brooklyn_client_path/"br/br"
+    end
+  end
+
+  test do
+    system "#{bin}/br", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!